npm_and_yarn/spec/dependabot/npm_and_yarn/file_fetcher_spec.rb

```diff
@@ -241,6 +241,23 @@
           to raise_error(Dependabot::PathDependenciesNotReachable)
       end
     end
+
+    context "with a path dependency that is an absolute path" do
```

Comment:
```suggestion
    context "with a path dependency that is a relative path pointing outside of the repo" do
```

I get that it's effectively an absolute path, but calling that felt slightly confusing...